src/app/pages/Settings.tsx

```diff
@@ -62,6 +65,18 @@ const TABS = [
       </>
     ),
   },
+  {
+    slug: "custom-networks",
+    title: "Custom networks",
+    Icon: (props: React.SVGProps<SVGSVGElement>) => <NetworkIcon {...props} className={clsx('fill-current', props.className)} />,
```

Comment:
Try to replace it with pure component: `NetworkIcon ` instead of component-factory. "fill-current" already exists inside I think:)

Reply:
And use `classNames` instead of `clsx` next time